themes-default/slim/src/components/show.vue

```diff
@@ -72,7 +72,7 @@ export default {
 
             // Not detailed
             if (!show.seasons) {
-                $store.dispatch('getShow', { id, indexer, detailed: true });
+                $store.dispatch('getShow', { id, indexer, detailed: false });
```

Comment:
@OmgImAlexis @sharkykh I need your help on this.
I'm trying to disable the detailed = true flag "for now". But it's causing an indefinite loop.
This made me try to understand how the show.vue component fits into all this. But I can't figure it out.

What i'm trying to accomplish is, to not use the series route with the detailed = true flag, to get the episodes, but to create alls on the /series/tvdb.../episodes route with the season flag, to get them per season.